wikimedia/wikipedia-ios
# [ALT TEXT] Complete UI for half sheet

**Phabricator:** https://phabricator.wikimedia.org/T370327

### Notes
This PR completes the UI for the half sheet.

### Test Steps
1. Enable all logging in WMFLogging.h
2. Fresh install, toggle on feature flags in Developer Settings, then go to Alt Text flow from Image Recommendations.
3. Confirm image, file name, character counter labels, and guidance labels display.
4. Type above 125 characters. Confirm character counter labels change and colors change to orange.
5. `character_warning` action should be logged to the console when warning appears.
6. Tap file name. `image_detail_view_click` action should be logged to the console.
7. (If going through live production edit) Tap on article image. Confirm gallery presents. Confirm when closing half sheet modal re-appears.
8. Confirm you cannot paste in the textview.
9. Go through the flow again with a device language set to es, pt, zh-hans, zh-hant or fr. Confirm localizations show on half sheet.
